src/blade/backend.py

```diff
@@ -37,10 +37,10 @@
 # NOTE the `$$` is required by ninja. and the `Multiple...` is the last and useless part of
 # the messages.
 _INCLUSION_STACK_SPLITTER = (r"awk '"
-    r"""/Multiple include guards may be useful for:/ {stop=1} """  # Can't exit here otherwise SIGPIPE maybe occurs.
-    r"""/^\.+ [^\/]/ { print $$0} """  # Non absolute path
-    r"""!/^\.+ / && !stop {print $$0 > "/dev/stderr"}"""  # Maybe error messages
-    r"'"
+     r"""/Multiple include guards may be useful for:/ {stop=1} """  # Can't exit here otherwise SIGPIPE maybe occurs.
+     r"""/^\.+ [^\/]/ && !stop { print $$0} """  # Non absolute path
+     r"""!/^\./ && !/^\// && !/Multiple include guards may be useful for:/ {print $$0 > "/dev/stderr"}"""  # Maybe error messages
```

Comment:
Suggest using the [next](https://www.gnu.org/software/gawk/manual/html_node/Next-Statement.html) command in the first line to avoid matching it again.